README.rst

```diff
@@ -25,7 +25,7 @@ Version Support
 ---------------
 
 *behave-django* is `tested against`_ the officially supported combinations of
-Python and Django (Django 3.2, 4.1, 4.2, 5.0 on Python 3.6, 3.7, 3.8, 3.9, 3.10, 3.11, 3.12).
+Python and Django (Django 3.2, 4.1, 4.2, 5.0 on Python 3.7, 3.8, 3.9, 3.10, 3.11, 3.12).
```

Comment:
heh. django 3.2 has a month and a half left of life https://endoflife.date/django

Reply:
I'd love to push a release to PyPI before Django 3.2 is obsoleted. There's still time to drop it afterwards.

The real problem is the dependency of behave-django [on a development release](https://github.com/behave/behave-django/blob/main/pyproject.toml#L52) of Behave, though. It makes use in air-gapped environments very difficult if not entirely impossible.